### PR TITLE
RA-981:Added The VisitType to the VisitContextModel class

### PR DESCRIPTION
**Pull request by tendomatGitHub**
https://issues.openmrs.org/browse/RA-981
Please Review this

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/coreapps/contextmodel/VisitContextModel.java [hunk 1]
```diff
@@ -23,7 +24,7 @@ public VisitContextModel(VisitDomainWrapper visit) {
         this.active = visit.isOpen();
         this.admitted = visit.isAdmitted();
         this.startDatetimeInMilliseconds = visit.getStartDatetime().getTime();
-
+        this.visitType = ConversionUtil.convertToRepresentation(visit.getVisit().getVisitType(), Representation.DEFAULT); // these are from the webservices.rest module
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/coreapps/contextmodel/VisitContextModel.java]: I do not think we need the comment that you have placed at the end.

**Comment by dkayiwa** [comment]
@tendomatGitHub did you follow the talk thread for this?

**Comment by tendomatGitHub** [comment]
yes and that it is related to the question i posed yesterday during scrum.

**Comment by dkayiwa** [comment]
@tendomatGitHub did you see Darius's recommendation on the talk thread?

**Comment by tendomatGitHub** [comment]
yes and that was my blocker....But let me now find out how best represent VisitTypt in {JSON REST representation}

**Comment by dkayiwa** [comment]
@tendomatGitHub ping me to reopen this when you are ready. :)

**Comment by tendomatGitHub** [comment]
ok sir

**Pull request closed without merging**